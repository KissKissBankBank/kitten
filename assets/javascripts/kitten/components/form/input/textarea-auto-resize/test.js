import React from 'react'
import renderer from 'react-test-renderer'
import { TextareaAutoResize } from './index'

describe('<TextareaAutoResize />', () => {
  let component

  describe('with default props', () => {
    beforeEach(() => {
      component = renderer.create(<TextareaAutoResize />).toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
