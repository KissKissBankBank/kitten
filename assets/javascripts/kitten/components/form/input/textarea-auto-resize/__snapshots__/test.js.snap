// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextareaAutoResize /> with default props matches with snapshot 1`] = `
<textarea
  onChange={[Function]}
  rows={1}
  style={
    Object {
      "lineHeight": "1.125rem",
    }
  }
/>
`;
